oceannavigator/frontend/src/components/lib/CheckBox.jsx

```diff
@@ -31,8 +31,8 @@ export default class CheckBox extends React.PureComponent {
 //***********************************************************************
 CheckBox.propTypes = {
   title: PropTypes.string.isRequired,
-  checked: PropTypes.bool.isRequired,
+  checked: PropTypes.bool,
```

Comment:
Was this causing an error? I would imagine it's necessary for the `CheckBox` component.

Reply:
Yes It causing an warning, because not all calling Checkbox component uses this checked option. Like below on Mapinput.jsx file >>>>>
                  <CheckBox
                    id='syncRanges'
                    onUpdate={this.props.changeHandler}
                    title={_("Sync Variable Ranges")}
                    style={{display: this.props.dataset_compare ? "block" : "none"}}
                  />